src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -500,9 +500,10 @@ public function assertJsonValidationErrors($keys)
     /**
      * Validate and return the decoded response JSON.
      *
+     * @param  string $key
```

Comment:
This should be `@param string|null $key` if I'm not wrong

Reply:
Oh thanks!